source/features/github-bugs.css

```
@@ -77,3 +77,19 @@
 .pagehead-actions :is(.btn, summary) .octicon {
 	margin-right: 4px !important;
 }
+
+/* Avoid excessive repo title wrap by deprioritizing the repo header actions #5398 */
+:root .pagehead-actions {
```

We still need a media query to keep the buttons hidden on mobile:

```suggestion
/* Avoid excessive repo title wrap by deprioritizing the repo header actions #5398 */
@media (min-width: 768px) {
	:root .pagehead-actions {
```